hbase098/README.md

```diff
@@ -1,5 +1,5 @@
 <!--
-Copyright (c) 2015 YCSB contributors. All rights reserved.
+Copyright (c) 2016 YCSB contributors. All rights reserved.
```

Comment:
I think this ought to be 2015-2016, since the original doc was still written in that year.